cmd/talosctl/cmd/talos/config.go

```diff
@@ -175,8 +179,58 @@ func openConfigAndContext(context string) (*clientconfig.Config, error) {
 	return c, nil
 }
 
+// configGetContexts represents config contexts command.
+var configGetContexts = &cobra.Command{
+	Use:    "contexts",
```

Comment:
`kubectl` calls this `kubectl config get-contexts` (idk if that is easier to distinguish from `context`)

Reply:
and at the same time `kubectl` has `use-context` to switch it, so it's a bit different naming scheme there.

Reply:
What do you think about renaming `context` to `use-context`? And then I can rename `contexts` to `get-contexts`.
_Or_ we can have aliases for that.

Reply:
we certainly could do aliases by the way with `context`/`contexts` being the defaults and k8s aliases secondary